BurnOutSharp/ProtectionType/Bitpool.cs

```diff
@@ -3,14 +3,35 @@
 
 namespace BurnOutSharp.ProtectionType
 {
+    /// <summary>
+    /// Bitpool is a copy protection that seems to resemble SafeDisc in several ways, found mostly in German releases.
+    /// Makes use of bad sectors, and contains what appears to be an encrypted game executable always called "CD.IDX" (similar to SafeDisc ICD files), which seems to be present in every disc protected with it.
+    /// The "CD.IDX" appears to purposefully contain bad sectors in most, if not all, cases.
+    /// A "bitpool.rsc" file is present in some, but not all Bitpool protected games. The purpose of it is so far unclear.
+    /// </summary>
     public class Bitpool : IPathCheck
     {
+
+        /// <summary>
+        /// Set of all ContentMatchSets for this protection
+        /// </summary>
+        private static readonly List<ContentMatchSet> contentMatchers = new List<ContentMatchSet>
```

Comment:
If you're adding content matches, you'll want to add `IContentCheck` to the inherited interfaces and implement the one method it requires.